ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -263,6 +263,19 @@ def quantile(input, q, dim=None, keepdim=False, *, interpolation="linear", out=N
 }
 
 
+@numpy_to_torch_style_args
+@to_ivy_arrays_and_back
+def nanquantile(input, q, dim=None, keepdim=False, *, interpolation="linear", out=None):
+    return ivy.nanquantile(
+        input, q, axis=dim, keepdims=keepdim, interpolation=interpolation, out=out
+    )
+
+
+nanquantile.unsupported_dtypes = {
+    "torch": ("float16", "bfloat16"),

```

Comment:
Actually can you add this as a decorator on the function.
```
@with_unsupported_dtypes(
    {"2.0.1 and below": ("float16", "bfloat16")}, "torch"
)
```
Sorry for making you change this the 2nd time 😅